### PR TITLE
add wrap for frozen

### PR DESCRIPTION
**Pull request by neheb**
Signed-off-by: Rosen Penev <rosenp@gmail.com>

### CODE REVIEW

**Comment by eli-schwartz** [comment]
Had to double-check the dependency name because upstream does not provide pkg-config, only cmake... dependency('frozen', method: 'cmake') does work though.

Would be nice for upstream to add a simple pkg-config file too. :)

**Comment by eli-schwartz** [comment]
Hmm, the tests/ directory should probably be added as test binaries for the meson port too...

**Comment by neheb** [comment]
Why?

**Comment by eli-schwartz** [comment]
This allows people to test that the header library works correctly for them while building their project. :) wraps do generally have the testsuites set up, sometimes guarded by a "tests" option in meson_options.txt though.

File: subprojects/packagefiles/frozen/LICENSE.build [hunk 1]
```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
```
Comment (xclaesse) [inline subprojects/packagefiles/frozen/LICENSE.build:4]: This should not be the license of the project, but for build file. I think we want to keep the same license for all wraps.

Comment (dcbaker) [reply subprojects/packagefiles/frozen/LICENSE.build:4]: I'm the past @jpakkane said they need to be the same as the project they're for, i think that's in the docs too

Comment (xclaesse) [reply subprojects/packagefiles/frozen/LICENSE.build:4]: Hum, could be worth clarifying this, I thought the intention was to keep them all under a permissive license. What if we have a wrap for a gpl project?

I was actually considering to set a single license file at the root of wraps, applying to the whole repository. Then release script can copy it in all patch tarballs automatically and we can stop duplicating it in packagefiles.

Comment (jpakkane) [reply subprojects/packagefiles/frozen/LICENSE.build:4]: The original reason for that was that we want to make it as easy as possible for projects to grab the wrapdb definitions and add them upstream.

Comment (neheb) [reply subprojects/packagefiles/frozen/LICENSE.build]: Replaced.

**Pull request merged**